src/broker/content-cache.c

```diff
@@ -703,6 +703,11 @@ static void content_unregister_backing_request (flux_t *h,
         flux_log_error (h, "error responding to unregister-backing request");
     if (cache->acct_dirty > 0)
         flux_log (h, LOG_ERR, "%d unflushables", cache->acct_dirty);
+    request_list_respond_error (&cache->flush_requests,
+                                cache->h,
+                                ENOSYS,
+                                NULL,
+                                "flush");
```

Comment:
Even though `request_list_respond_error()` is a no-op if there are no pending flush requests, having it run unconditionally looks suspicious.  Perhaps wrapping it in something like this would make it clearer:
```c
/* If backing store is unloaded with pending flush requests, ensure that
 * they receive an error response.
 */
if (cache->flush_requests != NULL) {
    request_list_respond_error (...);
}
```